deps.bzl

```diff
@@ -6969,12 +6969,12 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         urls = ["https://github.com/containerd/stargz-snapshotter/releases/download/v0.11.4/stargz-snapshotter-v0.11.4-linux-amd64.tar.gz"],
         build_file_content = 'exports_files(["stargz-store"])',
     )
+
     http_archive(
-        name = "com_github_firecracker_microvm_firecracker-v1.1.1-x86_64",
-        sha256 = "29194e13888358c6332995199ee288c081a8a23e68f0046944e4bb7f6548af57",
-        urls = ["https://github.com/firecracker-microvm/firecracker/releases/download/v1.1.1/firecracker-v1.1.1-x86_64.tgz"],
-        strip_prefix = "release-v1.1.1-x86_64",
-        build_file_content = 'exports_files(["firecracker-v1.1.1-x86_64", "jailer-v1.1.1-x86_64"])',
+        name = "com_github_maggie-lou_firecracker-v0.0.0-20230720-cf5f56f",
+        sha256 = "29f59ff51c554fca261980fec7845a9a6fa3db21f274e0a6f5e096cafc56c114",
+        urls = ["https://github.com/maggie-lou/firecracker/releases/download/v0.0.0-20230720-cf5f56f/firecracker-v0.0.0-20230720-cf5f56f.tgz"],
```

Comment:
I'd suggest copying the archive to the buildbuddy-tools GCS bucket rather than referencing your personal github account.

Reply:
Done